esmvaltool/utils/recipe_test_workflow/app/get_esmval/opt/rose-app-metoffice.conf

```
@@ -1,7 +1,15 @@
+# The reason behind the benign 'default' command below is because
+# if the '[command]' section does not exist
+# a '[WARN] command not defined' message is written to the 'job.err' file.
+
 [command]
-default=clone_latest_esmval.sh
+default=echo "Cloning ESMValCore and ESMValTool into ${ESMVAL_DIR}"
 
 [env]
-BRANCH=main
-ESMVALCORE_URL=git@github.com:ESMValGroup/ESMValCore.git
-ESMVALTOOL_URL=git@github.com:ESMValGroup/ESMValTool.git
+ROSE_FILE_INSTALL_ROOT=${ESMVAL_DIR}
+
+[file:ESMValCore]
```

```suggestion
[file:${ESMVALCORE_DIR}]
```
Single source of truthy goodness?

---

Nice! 🎉 I commit both suggestions, and also removed `ROSE_FILE_INSTALL_ROOT` in 467e13615, since it's not needed with the above change 👍